engine/runtime/src/main/java/org/enso/interpreter/node/expression/debug/EvalNode.java

```diff
@@ -85,8 +85,7 @@ RootCallTarget parseExpression(LocalScope scope, ModuleScope moduleScope, String
     var mod = newInlineContext.module$access$0().module$access$0();
 
     var m = org.enso.interpreter.runtime.Module.fromCompilerModule(mod);
-    var toTruffle =
-        new IrToTruffle(context, src, m.getScope(), compiler.org$enso$compiler$Compiler$$config);
+    var toTruffle = new IrToTruffle(context, src, m.getScope(), compiler.getConfig());
```

Comment:
I see. `getConfig()` is nicer. Can `org$enso$compiler$Compiler$$config` be made `private`, btw.?

Reply:
Seems that it can - 6abe0977bbba1d967a507d311deb8ed51342096c